tcms/rpc/tests/test_testcase.py

```diff
@@ -576,6 +749,10 @@ def verify_api_with_permission(self):
         self.assertEqual(result[2]['case_id'], self.case_3.pk)
         self.assertEqual(result[2]['sortkey'], 25)
 
+        # Test with query none
+        result = self.rpc_client.TestCase.sortkeys()
+        self.assertDictEqual(result, {})
```

Comment:
This is causing failures b/c your assertion is wrong. When the filter dict is empty all objects in the corresponding DB table will be returned. So that is all sortkeys for all test cases from all test plans that currently exist.

Reply:
Got it! I assumed it is `return {}`instead of `query = None`. :sweat_smile: